sig-network/README.md

```diff
@@ -35,8 +35,8 @@ The following subprojects are owned by sig-network:
 - **services**
   - Owners:
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/proxy/OWNERS
-    - https://raw.githubusercontent.com/kubernetes/pkg/kubernetes/master/controller/endpoint/OWNERS
-    - https://raw.githubusercontent.com/kubernetes/pkg/kubernetes/master/controller/service/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/controller/endpoint/OWNERS
```

Comment:
These two lines will also need to be changed in sigs.yaml, as they are generated from there.